src/main/java/digital/slovensko/autogram/core/errors/PkcsEidWindowsDllException.java

```diff
@@ -0,0 +1,7 @@
+package digital.slovensko.autogram.core.errors;
+
+public class PkcsEidWindowsDllException extends AutogramException {
+    public PkcsEidWindowsDllException(Exception e) {
+        super("Nastala chyba", "eID ovládač nie je možné použiť", "Nie je možné použiť eID ovládač pre podpisovanie občianskym preukazom. Pravdepodobne je potrebné do systému nainštalovať balík Microsoft Visual C++ 2015 <a href=\"https://www.google.com\">Redistributable</a>.\n\nAk to nepomôže, kontaktujte nás na podpora@slovensko.digital", e);
```

Comment:
Toto asi nutne nemusí byť len Eid ovládač. Taktiež sa mi úplne nezdá ten Google link. Nedáme tam tento? 

https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022

Prípadne ak sa bojíme že sa zmeni tak smerujeme 
na nejaký náš web kde bude redirect (jedno malé html na služby.slovensko.digital) to spraví.